src/OrchardCore.Modules/OrchardCore.Resources/Startup.cs

```diff
@@ -23,6 +31,8 @@ public override void ConfigureServices(IServiceCollection serviceCollection)
             });
 
             serviceCollection.AddTransient<IConfigureOptions<ResourceManagementOptions>, ResourceManagementOptionsConfiguration>();
+
+            serviceCollection.Configure<ResourceOptions>(settings => _configuration.GetSection("OrchardCore_Resources").Bind(settings));
```

Comment:
We still need to specify the parent section as we deal with an `IConfiguration` not an `IShellConfiguration`.

So normally `OrchardCore:OrchardCore_Resources`, I let you try it if necessary.

Reply:
Done